ontology/README.md

```diff
@@ -52,3 +52,40 @@ me:example:some_gene rdfs:seeAlso ENSEMBL:ESG00000XXXX . #and then pick up the a
 
 At this moment VG RDF wants a fully embedded variation graph. e.g. all positions in vg json have a single edit which covers a whole node. This is to enable easy SPARQL queries where substring operations are rarely used.
 
+
+## Annotations on pantograph
+
+On top of VG RDF, we can describe the same path information on Pantograph format as well.
+
+```ttl
+<pg/zoom10> a pg:ZoomLevel ;
+   pg:components <pg/zoom10/group1>, <pg/zoom10/group2> ;
+   pg:zoomFactor 10 .
+<pg/zoom1000> a pg:ZoomLevel ;
+   pg:components <pg/zoom1000/group1>, <pg/zoom1000/group2> ;
+   pg:zoomFactor 1000 . // zoomFactor is binWidth here.
+<pg/zoom1000/group1> a pg:Component ;
+   pg:componentRank 1 ;   # The order of component is inferred by rank.
+   pg:componentEdge <pg/zoom1000/group2> ;
+   pg:bins <pg/zoom1000/group1/bin1>, <pg/zoom1000/group1/bin2> ;
+   pg:steps <pg/zoom1000/group1/step1>, <pg/zoom1000/group1/step2> ;
+<pg/zoom1000/group1/step1> a pg:Step ;
+   pg:positionPercent 0.04
+   pg:inversionPercent 1
```

Comment:
Is inversionPercent 1 -> 100% or 1% ? as positionPercent is 0.04 I assume it was supposed to be 100% but something to clarify.

Reply:
Both inversionPercent and positionPercent are 1 -> 100%.
I just checked the code and example data.

Reply:
Sorry for setting an example value as 1, which causes ambiguity on whether the number is float or integer. We wish to regard the number as a float.